api/data_test.go

```diff
@@ -65,6 +67,36 @@ func TestGetCacheTimeEndpoint(t *testing.T) {
 	})
 }
 
+func TestGetCachetimeReturnsError(t *testing.T) {
+	Convey("When a non-existant cache time is requested with its ID", t, func() {
```

Comment:
These tests should be rewritten in a Given/When/Then style, to follow ONS' convention, which we're already doing in this file.

Reply:
We based the tests off of this file: https://github.com/ONSdigital/dp-dataset-api/blob/develop/api/dataset_test.go#L219